net.solarnetwork.node.weather.ibm.wc/OSGI-INF/blueprint/module.xml

```diff
@@ -0,0 +1,261 @@
+	<bean id="hourlyMessageSource" class="org.springframework.context.support.ResourceBundleMessageSource">
+		<property name="basenames">
+			<list>
+				<value>net.solarnetwork.node.weather.ibm.wc.MetserviceSupport</value>
+				<value>net.solarnetwork.node.weather.ibm.wc.WCHourlyDatumDataSource</value>
+			</list>
+		</property>
+	</bean>
+
+	<!-- support localized strings for the periodic job settings in the GUI -->
+	<bean id="dayJobMessageSource" class="net.solarnetwork.node.util.PrefixedMessageSource">
+		<property name="prefix" value="datumDataSource."/>
+		<property name="delegate">
+			<bean class="net.solarnetwork.node.util.PrefixedMessageSource">
+				<property name="prefix" value="delegate."/>
+				<property name="delegate">
+					<bean class="org.springframework.context.support.ResourceBundleMessageSource">
+						<property name="basenames" value="net.solarnetwork.node.support.WeatherLocationDatumDataSource"/>
+						<property name="parentMessageSource" ref="dayMessageSource"/>
+					</bean>
+				</property>
+			</bean>
+		</property>
+	</bean>
+	
+		<!-- support localized strings for the periodic job settings in the GUI -->
+	<bean id="hourlyJobMessageSource" class="net.solarnetwork.node.util.PrefixedMessageSource">
+		<property name="prefix" value="datumDataSource."/>
+		<property name="delegate">
+			<bean class="net.solarnetwork.node.util.PrefixedMessageSource">
+				<property name="prefix" value="delegate."/>
+				<property name="delegate">
+					<bean class="org.springframework.context.support.ResourceBundleMessageSource">
+						<property name="basenames" value="net.solarnetwork.node.support.WeatherLocationDatumDataSource"/>
+						<property name="parentMessageSource" ref="hourlyMessageSource"/>
+					</bean>
+				</property>
+			</bean>
+		</property>
+	</bean>
+
+	<!-- publish a "component factory" so our Plugin appears in the settings GUI -->
+	<bean id="daySettingsFactory" class="net.solarnetwork.node.settings.support.BasicSettingSpecifierProviderFactory">
+		<property name="displayName" value=" Weather Channel Plugin - Daily"/>
+		<property name="factoryUID" value="net.solarnetwork.node.weather.ibm.wc.day"/>
+		<property name="messageSource" ref="dayMessageSource"/>
+	</bean>
+		
+	<!-- publish a "component factory" so our Plugin appears in the settings GUI -->
+	<bean id="hourSettingsFactory" class="net.solarnetwork.node.settings.support.BasicSettingSpecifierProviderFactory">
+		<property name="displayName" value=" Weather Channel Plugin - Hourly"/>
+		<property name="factoryUID" value="net.solarnetwork.node.weather.ibm.wc.hour"/>
+		<property name="messageSource" ref="hourlyMessageSource"/>
+	</bean>
+	
+	<service ref="hourSettingsFactory" interface="net.solarnetwork.node.settings.SettingSpecifierProviderFactory"/>
+	<service ref="daySettingsFactory" interface="net.solarnetwork.node.settings.SettingSpecifierProviderFactory"/>
+
+	<bean id="objectMapper" class="net.solarnetwork.util.ObjectMapperFactoryBean"/>
```

Comment:
Instead of an `ObjectMapper` bean, we need a `BasicWCClient` bean so a single instance can be injected into all data source instances at runtime, e.g.

```xml
<bean id="basicClient" class="net.solarnetwork.node.weather.ibm.wc.BasicWCClient">
	<property name="objectMapper">
		<bean class="net.solarnetwork.util.ObjectMapperFactoryBean"/>
	</property>
</bean>
```